### ISSUE TITLE
Include TypeScript typings with highlight.js?

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [highlightjs/highlight.js#1650]
https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/highlight.js

It would be convenient if highlight.js provided these types as a part of the npm distribution. That way folks using TypeScript don't have to separately `npm i --save-dev @types/highlight.js`.

**Comment by jiehu567** [comment]
+1

**Comment by JoshuaKGoldberg** [comment]
Oh hey FWIW, since filing this post I've worked with some repos and come to the conclusion that it's generally better to not do what I posted here, and instead put them in DefinitelyTyped. Unless the repo is maintained by folks who are familiar with TypeScript _(in which case there's a good chance it would have been written in TypeScript to begin with 😉)_ having to package typing updates with the repo's maintenance becomes difficult. DefinitelyTyped has really nice systems for assigning reviewers and releasing.

Never mind the issue, will close. 😁
